tests/test/dashboard.test.js

```diff
@@ -5,17 +5,22 @@ describe('The Unlock Dashboard', () => {
     await page.goto(url('/dashboard'))
   })
 
+  const testLockAddress = '0x4983D5ECDc5cc0E499c2D23BF4Ac32B982bAe53a';
```

Comment:
Maybe add a comment stating that the locks are always deployed at the same addresses?